test/e2e/windows/hybrid_network.go

```diff
@@ -79,6 +72,28 @@ var _ = SIGDescribe("Hybrid cluster network", func() {
 
 		})
 
+		ginkgo.It("should provide Internet connection for containers using DNS [Feature:Networking-DNS]", func() {
+			linuxPod := createTestPod(f, linuxBusyBoxImage, linuxOS)
+			ginkgo.By("creating a linux pod and waiting for it to be running")
+			linuxPod = f.PodClient().CreateSync(linuxPod)
+
+			ginkgo.By("verifying pod external connectivity to the internet")
+
+			ginkgo.By("checking connectivity to 8.8.8.8 53 (google.com) from Linux")
+			assertConsistentConnectivity(f, linuxPod.ObjectMeta.Name, linuxOS, linuxCheck("www.google.com"))
```

Comment:
you've changed the linuxCheck line
```
			assertConsistentConnectivity(f, linuxPod.ObjectMeta.Name, linuxOS, linuxCheck("8.8.8.8", 53))
```

it has a different signature, requires name and port